test/tests/divertor_monoblock/divertor_monoblock.i

```diff
@@ -683,18 +695,24 @@ tungsten_atomic_density = ${units 6.338e28 m^-3}
 [Functions]
     ### Maximum mobile flux of 7.90e-13 at the top surface (1.0e-4 [m])
     ### 1.80e23/m^2-s = (5.0e23/m^2-s) *(1-0.999) = (7.90e-13)*(${tungsten_atomic_density})/(1.0e-4)  at steady state
+    [plasma_time_func]
```

Comment:
```suggestion
    [plasma_time_function]
```
I prefer to avoid abbreviations. It is usually more confusing than anything else.

Reply:
I I know you are just being consistent with the other function names, but I'll update that in my clean up.